src/ResultsActivity/elements/PayOptions/PayOptions.js

```diff
@@ -79,17 +78,18 @@ function PayOptions({ offenderDetails, updateOffenderAttendance, date, openModal
           </Option>
         </Fragment>
       )}
-
       <Option data-qa="other-option" className="row-gutters">
-        {showRadioButton && (
-          <Radio name={offenderNo} onChange={renderForm} value="other" checked={selectedOption === 'other'}>
-            <VisuallyHidden>Other</VisuallyHidden>
-          </Radio>
-        )}
+        {showRadioButton &&
+          !absentReason && (
+            <Radio name={offenderNo} onChange={renderForm} value="other" checked={selectedOption === 'other'}>
+              <VisuallyHidden>Other</VisuallyHidden>
+            </Radio>
+          )}
         {allowUpdate &&
-          other && (
-            <UpdateLink onClick={renderForm}>
-              <OtherMessage data-qa="other-message">View/ Update</OtherMessage>
+          other &&
+          absentReason && (
```

Comment:
`allowUpdate` might be better containing `other` and `absentReason` too

Reply:
I like it!